src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/DataClassification/SensitivityClassification.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.Data.SqlClient.DataClassification
+{
+    /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/Label.xml' path='docs/members[@name="Label"]/Label/*' />
+    public class Label
+    {
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/Label.xml' path='docs/members[@name="Label"]/Name/*' />
+        public string Name { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/Label.xml' path='docs/members[@name="Label"]/Id/*' />
+        public string Id { get; private set; }
+
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/Label.xml' path='docs/members[@name="Label"]/ctor/*' />
+        public Label(string name, string id)
+        {
+            Name = name;
+            Id = id;
+        }
+    }
+
+    /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/InformationType.xml' path='docs/members[@name="InformationType"]/InformationType/*' />
+    public class InformationType
+    {
+        /// <include file='..\../../../../../../doc/snippets/Microsoft.Data.SqlClient.DataClassification/InformationType.xml' path='docs/members[@name="InformationType"]/Name/*' />
```

Comment:
Please change the single back slash to slash. this will prevent the code from compiling on linux.